src/class.ziprecipes.php

```diff
@@ -298,8 +298,8 @@ public static function zrdn_format_recipe($recipe) {
 				'print_permalink_hide' => get_option('zlrecipe_printed_permalink_hide'),
 				'copyright' => get_option('zlrecipe_printed_copyright_statement')
 		);
-
-		return Util::view("recipe", $viewParams);
+        $custom_template = apply_filters('zrdn__custom_templates_show_tpl', false, $viewParams);
+        return $custom_template ? $custom_template : Util::view('recipe', $viewParams);
```

Comment:
Can be shortened to:

return $custom_template ?: Util::view('recipe', $viewParams);

[SO explanation](http://stackoverflow.com/questions/1276909/php-syntax-question-what-does-the-question-mark-and-colon-mean#1276912)